src/main/java/com/github/rjeschke/txtmark/DefaultDecorator.java

```diff
@@ -241,4 +241,28 @@ public void closeImage(final StringBuilder out)
     {
         out.append(" />");
     }
+
+    /** @see com.github.rjeschke.txtmark.Decorator#openUnderlined(StringBuilder) */
+	@Override
+	public void openUnderlined(StringBuilder out) {
```

Comment:
I don't remember me using this bracing style anywhere in any of my Java code files.